autoload/nvimdev.vim

```diff
@@ -56,15 +56,15 @@ function! nvimdev#init(path) abort
   endif
 
   " Use with :Neomake! make
+  " make will try to build  both dependancies and nvim
+  " Ignore entering .deps directory not to mess the quickfix-dirstack
+  let s:tmp_efm = '%-Gninja: Entering directory `.deps'','
+  let s:tmp_efm .= &efm
```

Comment:
What is the value of `errorformat` that's being appended here?  Is this relying on the one being set by Neomake?